pkgs/development/interpreters/tclreadline/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv
+, fetchFromGitHub
+, automake
+, autoconf
+, libtool
+, readline
+, tcl
+, tk
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tclreadline";
+  version = "2.3.6";
+
+  src = fetchFromGitHub {
+    owner = "flightaware";
+    repo = "tclreadline";
+    rev = "v${version}";
+    sha256 = "0ik2nr6c9icgmx98zn2vginc89l87xn2d9rmxqjgpqpyfmj14dc3";
+  };
```

Comment:
```suggestion
  };
  
  patches = [
    # do not run ./configure from ./autogen.sh
    (fetchpatch {
      url = "https://github.com/flightaware/tclreadline/commit/1be33100816577058cb01e74f1bd5abf4c5ae438.patch";
      sha256 = "1mh96sdcr0kz0j41pn2873j3xc9d7cbxnq8h1mywwxrqffvkj44v";
    })
  ];
```

Reply:
They merged your pr and made a point release. I'm going to update to that